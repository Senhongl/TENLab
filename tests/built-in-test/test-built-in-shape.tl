a = [[2,3,4,5],[5,1,4,3],[2,3,1,3]];
b = shape(a);
print(b);
