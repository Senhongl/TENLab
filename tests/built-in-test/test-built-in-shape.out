 3
 4
[ CPUIntType{2} ]
